server/game/SimultaneousEvents.js

```diff
@@ -9,6 +9,7 @@ class SimultaneousEvents {
     }
 
     addChildEvent(event) {
+        event.parentEvent = this;
```

Comment:
Can't remember if not having this before was an oversight, or if SimultaneousEvent is special in some way and adding this will Cause Problems ™️

Reply:
This is, currently, just used as part of an adjustment for **Tywin Lannister (LoCR)** to ensure that only 1 card is being discarded simultaneously. Now that I think about it, it could easily be an aggregated event, and checking if the `event.length === 1`; not entirely sure why I didn't go with that. I'll consider, and maybe revert (with Tywin changes)